conf.py

```diff
@@ -246,21 +260,9 @@
     {
         "name": "nifi",
         "versions": [
-            {
-                "product": "1.21.0",
-                "java-base": "11",
-                "java-devel": "11"
-            },
-            {
-                "product": "1.23.2",
-                "java-base": "11",
-                "java-devel": "11"
-            },
-            {
-                "product": "1.25.0",
-                "java-base": "21",
-                "java-devel": "11"
-            },
+            {"product": "1.21.0", "java-base": "11", "java-devel": "11"},
+            {"product": "1.23.2", "java-base": "11", "java-devel": "11"},
+            {"product": "1.25.0", "java-base": "21", "java-devel": "11"},
```

Comment:
What are the reasons for this :) ?

Reply:
talk to my editor :)

Reply:
Hello Mr. Editor, what are the reasons for this :) ?